js/browzine.js

```diff
@@ -7,44 +7,62 @@
       $sceDelegateProvider.resourceUrlWhitelist(urlWhitelist);
   }]);
 
-// Add Browzine Links
-  app.controller('prmBriefResultAfterController', function($scope, $http) { 
+// Add Article In Context & Browzine Links
+  app.controller('prmSearchResultAvailabilityLineAfterController', function($scope, $http) { 
     var vm = this;
-    if (vm.parentCtrl.item.pnx.addata.doi)  {
-          vm.doi = vm.parentCtrl.item.pnx.addata.doi[0] || '';
-          var url = "https://yourserver.edu/primo/browzine/browzineArticleInContext?DOI=" + vm.doi;
-          $http.jsonp(url, {jsonpCallbackParam: 'callback'}).then(function(response) {
-          $scope.article = response.data;
-          }, function(error){
-                 console.log(error); //
+		if (vm.parentCtrl.result.pnx.addata.doi && vm.parentCtrl.result.pnx.display.type[0] == 'article')	{
+        	vm.doi = vm.parentCtrl.result.pnx.addata.doi[0] || '';
+        	var articleURL = "https://yourserver.edu/primo/browzine/browzineArticleInContext?DOI=" + vm.doi;
+        	$http.jsonp(articleURL, {jsonpCallbackParam: 'callback'}).then(function(response) {
+        	  $scope.article = response.data;
+        	}, function(error){
+            console.log(error);
             });
-      }
-
+    	}
+      if (vm.parentCtrl.result.pnx.addata.issn && vm.parentCtrl.result.pnx.display.type[0] == 'journal')	{	
+        	vm.issn = vm.parentCtrl.result.pnx.addata.issn[0].replace("-", "") || '';
+        	var journalURL = "https://yourserver.edu/primo/browzine/browzineJournals?ISSN=" + vm.issn;
```

Comment:
Since `yourserver.edu` is sprinkled throughout this file, would it make sense to define it once up at the top, then use string literals throughout the rest of the file, to reduce the likelihood of typos?

Something like:
```
const serverName = 'yourserver.edu';
...
var journalURL = `https://${serverName}/primo/browzine/browzineJournals?ISSN=${vm.issn}`;
```

perhaps?  Changing yourserver.edu becomes a one-line change instead of a multi-line find and replace in this case.